app/templates/components/location-map.hbs

```diff
@@ -1 +1 @@
-{{yield}}
+<div class="map-container" aria-hidden="true" tabindex="-1"></div>
```

Comment:
@jenweber question about this: we are also assigning the "map-container" class to the parent element (see location-map.js) and actually assigning the map element to the components base element.  I don't understand why we need a second div with the "map-container" class.  Should we rather assign the aria-hidden and tabindex attributes to the component element itself?

Reply:
Whichever you think is best! It was probably just an oversight to have duplicates.